assets/wizards/advertising/views/ad-units/index.js

```diff
@@ -5,97 +5,168 @@
 /**
  * WordPress dependencies
  */
+import { useState, Fragment } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
-
-/**
- * External dependencies
- */
-import classnames from 'classnames';
+import { trash, pencil } from '@wordpress/icons';
 
 /**
  * Internal dependencies
  */
-import { ActionCard, Notice, withWizardScreen } from '../../../../components/src';
+import {
+	ActionCard,
+	TextControl,
+	Button,
+	Card,
+	Notice,
+	withWizardScreen,
+} from '../../../../components/src';
 
 /**
  * Advertising management screen.
  */
-class AdUnits extends Component {
-	/**
-	 * Render.
-	 */
-	render() {
-		const { adUnits, onDelete, service, isGAMConnected } = this.props;
+const AdUnits = ( {
+	adUnits,
+	onDelete,
+	updateAdUnit,
+	wizardApiFetch,
+	service,
+	gamConnectionStatus = {},
+} ) => {
+	const warningNoticeText = `${ __(
+		'Please connect your Google account using the Newspack dashboard in order to use ad units from your GAM account.',
+		'newspack'
+	) } ${
+		Object.values( adUnits ).length
+			? __( 'The legacy ad units will continue to work, but they are not editable.', 'newspack' )
```

Comment:
Is this true anymore? Aren't legacy units now editable?

Reply:
Good catch, fixed in bb65fde8276a12fd3e56c6c98c1a0cf5380c9ff0